spec/support/landing_folder_support/file_objects/et1_pdf_file.rb

```diff
@@ -0,0 +1,71 @@
+require 'rspec/matchers'
+require_relative './base_pdf_file'
+
+module EtApi
+  module Test
+    module FileObjects
+      # Represents the ET3 PDF file and provides assistance in validating its contents
+      class Et1PdfFile < BasePdfFile
+        def has_correct_contents_for?(claim:, claimants:, respondents:, representative:) # rubocop:disable Naming/PredicateName
+          Et1PdfFileSection::YourDetailsSection.new(form, lookup_root, template: template).has_contents_for?(claimant: claimants.first)
```

Comment:
As a performance boost - we now pass the form (which is the parsed pdf form) to each section rather than each section calling pdftk library itself.